frontend/test/metabase/scenarios/binning/correctness/longitude.cy.spec.js

```diff
@@ -0,0 +1,110 @@
+import { restore, popover, openPeopleTable } from "__support__/e2e/cypress";
+
+describe("scenarios > binning > correctness > longitude", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+    openPeopleTable();
+    cy.findByText("Summarize").click();
+  });
+
+  it("Auto bin", () => {
+    openPopoverFromSelectedBinningOption("Longitude", "Auto bin");
+    popover().within(() => {
+      cy.findByText("Auto bin").click();
+    });
+    getTitle("Count by Longitude: Auto binned");
+    cy.get(".bar");
+    cy.get(".y-axis-label")
+      .invoke("text")
+      .should("eq", "Count");
+    cy.get(".x-axis-label")
+      .invoke("text")
+      .should("eq", "Longitude");
+
+    cy.get(".axis.x").within(() => {
+      cy.findByText("170° W");
```

Comment:
@flamber @ariya - I was thinking to literary pass the array of all X axis "ticks" for each of the binning buckets to make sure they render correctly.
Do you think it's an overkill?

Alternatively, I can pick just a few representative values and make sure they render.

Reply:
I would just pick the `100°` tick and then get the tick before and after. Or the first three ticks, but at 1° might skip some degrees

Reply:
I agree with @flamber, just sample some ticks.

Reply:
This was resolved. I'll mark the discussion as such.
Thanks